store/copr/batch_coprocessor.go

```diff
@@ -1169,3 +1177,92 @@ func (b *batchCopIterator) handleCollectExecutionInfo(bo *Backoffer, resp *batch
 	}
 	resp.detail.CalleeAddress = task.storeAddr
 }
+
```

Comment:
Just copied [ buildBatchCopTasksConsistentHash](https://github.com/pingcap/tidb/blob/d4e2b2eac1c93cef247ebb7d0bdff83474eda94a/store/copr/batch_coprocessor.go#L553) before this pr:  buildBatchCopTasksConsistentHash